.github/workflows/sonarqube.yml

```diff
@@ -2,27 +2,65 @@ name: Sonarqube
 
 on:
   pull_request:
-      types: [opened, synchronize, reopened]
+    types: [opened, synchronize, reopened]
+    branches:
+      - development
```

Comment:
Lets use same pipeline for main branch as well

branches: [ development, main]